src/merkle/smt/full/mod.rs

```diff
@@ -25,6 +25,15 @@ pub const SMT_DEPTH: u8 = 64;
 // SMT
 // ================================================================================================
 
+/// Sparse Merkle tree mapping 256-bit keys to 256-bit values. Both keys and values are represented
+/// by 4 field elements.
+///
+/// All leaves sit at depth 64. The most significant word of the key is used to identify the leaf to
+/// which the key maps.
```

Comment:
nit: probably should be "The most significant element of the key..." (instead of "The most significant word of the key...").